### DISCUSSION TITLE
A new idea to group the classes for different screen sizes - Syntactical Sugar

### DISCUSSION THREAD

**Discussion by yasharya2901** [tailwindlabs/tailwindcss#15473] [Ideas]
Recently, I faced a challenge where I had to make the website responsive. The website was designed for the desktop only. It had a lot of classes. So, I what I did was added `md:` prefix in front of all the classes, and started adding new class for the mobile screen. However, if it had supported the syntax like `className=md:(flex flex-1 pd-2)`, it would be very beneficial to group classes suitable for different screen. Or, at least, I have a set of class I want to apply to medium and large screens. Just a syntactical sugar.

What do you say?

**Comment by wongjn** [comment]
There have been suggestions about adding such a feature. The maintainers have addressed this request before, that you may wish to read up on if you have not already done so:
- [A Twitter thread from the creator of Tailwind, Adam Wathan](https://twitter.com/adamwathan/status/1461519820411789314)
- [A previous GitHub discussion thread with maintainer responses](https://github.com/tailwindlabs/tailwindcss/discussions/8337#discussioncomment-4032611)

**Comment by hasnain-neosoft** [comment]
className=md:(flex flex-1 pd-2)
useful
